script/installer.js

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env node
+
+const createWindowsInstaller = require('electron-winstaller').createWindowsInstaller
+const path = require('path')
+const rimraf = require('rimraf')
+
+deleteOutputFolder()
+  .then(getInstallerConfig)
+  .then(createWindowsInstaller)
+  .catch((error) => {
+    console.error(error.message || error)
+    process.exit(1)
+  })
+
+function getInstallerConfig () {
+  const rootPath = path.join(__dirname, '..')
+  const outPath = path.join(rootPath, 'release')
+
+  return Promise.resolve({
+    appDirectory: path.join(outPath, 'Mattermost-win32-ia32'),
+    iconUrl: 'https://raw.githubusercontent.com/mattermost/desktop/master/resources/icon.ico',
+    //loadingGif: path.join(rootPath, 'assets', 'img', 'loading.gif'),
```

Comment:
It would be nice if you could get somebody to provide a now installing splash screen. See https://github.com/electron/electron-api-demos/blob/master/assets/img/loading.gif as an example